pkgs/http/lib/retry.dart

```diff
@@ -14,7 +14,7 @@ import 'http.dart';
 /// NOTE: [RetryClient] makes a copy of the request data in order to support
 /// resending it. This can cause a lot of memory usage when sending a large
 /// [StreamedRequest].
-class RetryClient extends BaseClient {
+final class RetryClient extends BaseClient {
```

Comment:
Interface is identical to client. There is no reason to implement or extend this.